src/modules/fw_pos_control/FixedwingPositionControl.cpp

```diff
@@ -3047,6 +2940,17 @@ fw_pos_control is the fixed-wing position controller.
 
 	return 0;
 }
+float FixedwingPositionControl::getLoadFactor()
+{
+	float load_factor_from_bank_angle = 1.0f;
+
+	if (PX4_ISFINITE(_att_sp.roll_body)) {
+		load_factor_from_bank_angle = 1.0f / cosf(_att_sp.roll_body);
```

Comment:
Do we need to guard it against negative load factors?

Reply:
well, at least constrain the cos(roll) > 0 (same as to guard against +/-90deg roll, since we anyway dont really handle upsides down flight in the FW controller). 

good catch @sfuhrer

Reply:
Done